src/Hangfire.PostgreSql/Connectivity/DefaultConnectionBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Npgsql;
+
+namespace Hangfire.PostgreSql.Connectivity
+{
+    public class DefaultConnectionBuilder : IConnectionBuilder
```

Comment:
DefaultConnectionBuilder can be made `internal sealed` in order to not expose redundant class outside.

Reply:
This was intentional, as I consider `IConnectionBuilder` an extensibility point, users may want to use `DefaultConnectionBuilder` as a base for a custom builder.